src/libs/EmojiUtils.js

```diff
@@ -70,6 +70,34 @@ function isSingleEmoji(message) {
     return matchedUnicode === currentMessageUnicode;
 }
 
+/**
+ * Validates that this message contains only emojis
+ *
+ * @param {String} message
+ * @returns {Boolean}
+ */
+function containsOnlyEmojis(message) {
+    const match = message.replace(/ /g, '').replaceAll('\n', '').match(CONST.REGEX.EMOJIS);
+
+    if (!match) {
+        return false;
+    }
+
+    const codes = [];
+    _.map(match, emoji => _.map(getEmojiUnicode(emoji).split(' '), (code) => {
+        if (code !== CONST.EMOJI_INVISIBLE_CODEPOINT) {
+            codes.push(code);
+        }
+        return code;
+    }));
+
+    // Emojis are stored as multiple characters, so we're using spread operator
+    // to iterate over the actual emojis, not just characters that compose them
+    const messageCodes = _.filter(_.map([...message.replace(/ /g, '').replaceAll('\n', '')],
```

Comment:
This is not DRY, `message.replace(/ /g, '').replaceAll('\n', '')` computing this twice.

Reply:
> This is not DRY, `message.replace(/ /g, '').replaceAll('\n', '')` computing this twice.

All done